docs/handlers.md

```diff
@@ -0,0 +1,118 @@
+# Handler plugin development
+
+The purpose of Handler plugins is to generate some kind of output from log entries.
+This can mean writing some representation of the entries into a log file, or sending
+the raw entries to an external service for further processing, or anything else, really.
+
+## The `HandlerPlugin` interface
+
+Each Handler plugin must implement the `HandlerPlugin` interface, which extends
+the generic [`Plugin` interface] and adds the following methods and properties:
+
+ - `public log(entry: LogEntry): Promise<void> | void`: This method will be called from
+   `Logger.add()`. Whatever a handler wants to do with log entries is up to the handler.
```

Comment:
"Whatever a handler wants to do with log entries is up to the handler" Tohle je trosku tricky a mame to jinak nez jine loggery. Trosku bych to zduraznil a rozepsal.

Reply:
tomu uplne nerozumim, co ti po precteni tyhle vety neni jasny? na nekolika mistech se pise, ze handlery bud posilaj entry na nejakej externi endpoint, nebo je nejak zformatujou a nekam zapisou, a navic tohle je chovani tech nasich vestavenejch handleru, 3rd party handlery muzou delat jeste X jinejch veci

Reply:
Casto logger odlisuje kam poslat jaky log. A pak handler rovnou zpracovava, co dostane.
U nas logger posila vsechno vsude. Handler si pak filtruje co zpracuje nebo ne. Tohle mame odlisne a je dulezite to zduraznit

Reply:
jo aha, už chápu